crates/re_query_cache/src/cache.rs

```diff
@@ -333,29 +333,17 @@ impl CacheBucket {
 // For this reason we move as much of the code as possible into the already existing macros in `query.rs`.
 
 /// Caches the results of `LatestAt` queries.
```

Comment:
Expand on this please - are these single component queries, or archetype queries? Are we expected to have one `LatestAtCache` for the entire store, or multiple ones?